root/etc/e-smith/templates/etc/httpd/conf.d/tancredi.conf/10base

```diff
@@ -0,0 +1,26 @@
+{
+    use esmith::NetworksDB;
+
+    my $ndb = esmith::NetworksDB->open_ro();
+    my @localAccessSpec = $ndb->local_access_spec();
+    our $localAccess = join(' ',@localAccessSpec);
+
+    our $validFrom = ${'httpd-fpbx'}{'ValidFrom'};
+    $validFrom =~ s/,/ /g;
+
+    $OUT = "";
+}
+
+<LocationMatch "/provisioning/[a-zA-Z0-9\.\-_]+$">
+    Require ip {$localAccess} {$validFrom}
```

Comment:
Does Apache need a ProxyPass directive here? I'm afraid the next Location stanza is not evaluated if the regexp matches...

Reply:
This configuration works:
```
<LocationMatch "/provisioning/[a-zA-Z0-9\.\-_]+$">
    Require ip 127.0.0.1 192.168.122.75
</LocationMatch>

<Location "/provisioning">
    ProxyPass "fcgi://127.0.0.1:9605/usr/share/tancredi/public/provisioning.php"
</Location>
```

This doesn't:
```
<LocationMatch "/provisioning/[a-zA-Z0-9\.\-_]+$">
    Require ip 127.0.0.1 192.168.122.75
    ProxyPass "fcgi://127.0.0.1:9605/usr/share/tancredi/public/provisioning.php"
</LocationMatch>

<Location "/provisioning">
    ProxyPass "fcgi://127.0.0.1:9605/usr/share/tancredi/public/provisioning.php"
</Location>
```
It fails with error `ProxyPass/<Proxy> and ProxyPassMatch/<ProxyMatch> can't be used altogether with the same worker name (fcgi://127.0.0.1:9605/usr/share/tancredi/public/provisioning.php)` 
but httpd documentation isn't really clear to me https://httpd.apache.org/docs/current/mod/core.html#locationmatch